web-components/src/[sandbox]/examples/draggable.ts

```diff
@@ -0,0 +1,288 @@
+import "@/components/draggable/Draggable";
+import "@/components/list/List";
+import "@/components/list/ListItem";
+import "@/components/checkbox/Checkbox";
+import "@/components/checkbox/CheckboxGroup";
+import "@/components/icon/Icon";
+import { comboBoxOptions } from "@/[sandbox]/sandbox.mock";
+import reset from "@/wc_scss/reset.scss";
+import { List } from "@/components/list/List";
+import { ListItem } from "@/components/list/ListItem";
+import { css, customElement, html, LitElement, query, queryAll } from "lit-element";
+import Sortable, { GroupOptions } from "sortablejs";
+
+@customElement("default-draggable-sandbox")
+export class DefaultDraggable extends LitElement {
+  @query("md-list") list!: List.ELEMENT;
```

Comment:
We updated this pattern correct? Take Alert for example:
```
 @customElementWithCheck("md-alert")
  export class ELEMENT extends LitElement {
  ```
  to get the element checks from #766

Reply:
Thx @futurethang. Fixed